assets/js/blocks/add-to-cart-form/edit.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Disabled,
+	Notice,
+	__experimentalInputControl as InputControl,
```

Comment:
Just saw the warning regarding the usage of experimental components: I'm replacing it.